2.2-taptweak.ipynb

```diff
@@ -6,7 +6,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import hashlib\n",
     "from io import BytesIO\n",
     "import random\n",
     "\n",
```

Comment:
Missed import of sha256 on `from test_framework.messages` line

Reply:
fixed